src/cmd/devdraw/cocoa-screen-metal.m

```diff
@@ -217,6 +217,7 @@ + (void)makewin:(NSValue *)v
 	layer.pixelFormat = MTLPixelFormatBGRA8Unorm;
 	layer.framebufferOnly = YES;
 	layer.opaque = YES;
+	layer.colorspace = CGColorSpaceCreateWithName((__bridge CFStringRef)NSDeviceWhiteColorSpace);
```

Comment:
1. I don't know what I'm doing with object ownership / memory management here, but I think NSDeviceWhiteColorSpace is a global so we don't have to become owner of it
2. It seems awkward to have to use a "NS" constant as an input to a "CG" function. Maybe there is a better way?

Reply:
3. I also can't explain why what I am doing here is "right", it just seems to work. This uses the same color space we use when instantiating the cursor images.